src/cr-logging/Logging.cs

```diff
@@ -0,0 +1,108 @@
+﻿// <copyright file="Logging.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging
+{
+    using System;
+    using System.Configuration;
+    using Microsoft.Extensions.Configuration;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using Serilog.Formatting;
+
+    /// <summary>
+    /// Main Logging class. Holds the methods for setting up a SeriLog Logger.
+    /// </summary>
+    public static class Logging
+    {
+        private static IConfigurationRoot _appConfig;
+
+        private delegate bool TryParse<in T1, T2>(T1 valueToParse, out T2 obj2);
+
+        /// <summary>
+        /// Function to setup the SeriLog Logger.
```

Comment:
How does it setup the SeriLog logger, what details does it use, etc. What types of logger does it support in our implementation.